core/src/context.rs

```diff
@@ -10,6 +10,13 @@ use std::sync::{
     Arc, Mutex, RwLock, RwLockReadGuard,
 };
 
+use holochain_cas_implementations::cas::file::FilesystemStorage;
+
+#[cfg(target_os = "linux")]
```

Comment:
@AshantiMutinta does this target mac as well?

Reply:
Oh yes, will be having a push in a few minutes

Reply:
There is already a crate for this that manages home directories

Reply:
Specifically: https://crates.io/crates/dirs

Reply:
Or: https://crates.io/crates/directories

Reply:
Nice, will make those updates and push! Thank you

Reply:
I don't think this belongs in here. The path should be set in the container. Ultimately, the CAS implementation has to be instantiated in the container and passed in to `holochain_new()` via the context. But we said for the DevCamp we always construct a `FileSystemStorage` in `core_api` with only the path set by the container.

In HoloSqape (and hcshell of that matter) I am using Qt's `QStandardPaths` to get the right path for the settings, which are actually very different for all three OSs (like $HOME/Library/Preferences/<App Name> for macOS and something else for Windows)